### ISSUE TITLE
Make downloads easier for older versions of Android

### ISSUE THREAD

**Issue by alexfigtree** [openwallet-foundation-labs/learner-credential-wallet#663]
**Is your feature request related to a problem? Please describe.**
Current minSdkVersion for Android is at 34, which means that those with older Android phones that have an older older OS cannot download the LCW.

**Describe the solution you'd like**
Lower the minSdkVersion so that the LCW is compatible with older versions of Android

**Describe alternatives you've considered**
Upgrading OS is not always possible

**Additional context**
A couple of requests have been made for compatibility with Android 10, therefore, we should lower the minSdkVersion in app.json to 29.

**Comment by alexfigtree** [comment]
Resolved in https://github.com/openwallet-foundation-labs/learner-credential-wallet/pull/662
